src/routes/bestPractice.router.js

```diff
@@ -0,0 +1,12 @@
+import { Router } from 'express';
+import createBestPractice from '../controllers/bestPractices.controller';
+
+const router = Router();
+
+router.post('/submit-form', (req, res, next) => {
+  createBestPractice({ ...req.body });
+  res.json({ success: 'action posted' });
+  next();
```

Comment:
Nice job, but handle all the application logic on the controller, and let the router to only deals with routing.

Reply:
Thanks. I'll move the logic to the controller and it'll be an import only in the router.

Reply:
yeah, exactly

Reply:
Fixed.